server/models/plugins/stripe-customer.js

```diff
@@ -54,7 +54,7 @@ module.exports = exports = function stripeCustomer (schema, options) {
         user.stripe.customerId = customer.id;
       }
 
-      var card = customer.cards.data[0];
+      var card = customer.cards.data[0] || customer.sources.data[0];
```

Comment:
I missed this. it should be `var cardSource = customer.cards || customer.sources;`, then `var card = cardSource.data[0]`